application/shared-kernel/ApiCore/Endpoints/TrackEndpoints.cs

```diff
@@ -0,0 +1,195 @@
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Routing;
+using StackFrame = Microsoft.ApplicationInsights.DataContracts.StackFrame;
+
+namespace PlatformPlatform.SharedKernel.ApiCore.Endpoints;
+
+public static class TrackEndpoints
+{
+    public static void MapTrackEndpoints(this IEndpointRouteBuilder routes)
+    {
+        routes.MapPost("/api/track", Track);
+    }
+
+    private static TrackResponseSuccessDto Track(
+        List<TrackRequestDto> trackRequests,
+        TelemetryClient telemetryClient,
+        ILogger<TrackEndpointsLoggerCategory> logger
```

Comment:
Change this to a generic logger instead of creating a new class.


```suggestion
        ILogger<string> logger
```